package.json

```diff
@@ -23,13 +23,14 @@
     "deploy:doc": "git-directory-deploy --directory build/ --branch gh-pages",
     "prebuild:css": "mkdir dist/",
     "prebuild:css:app": "mkdir -p build/styleguide",
-    "build:css": "stylus -c --include stylus -o dist/cozy-ui.min.css stylus/cozy-ui/build.styl",
-    "build:css:app": "stylus --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl",
+    "build:css": "stylus -c --include stylus -o dist/cozy-ui.min.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
+    "build:css:app": "stylus --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
```

Comment:
At this point, it may be more readable to use the JS API instead of the CLI -> move the logic to a dedicated script ?